io/net/vlan_test.py

```diff
@@ -222,12 +225,14 @@ def test_vlan_1_2230(self):
                     Now ping. it should FAIL
         """
         self.vlan_port_conf("1", "2230")
+        time.sleep(5)
```

Comment:
why sleep ?

Reply:
Is this needed ?

Reply:
can we have some sysfs check to achieve this sleep ?

Reply:
sysfs check ..?which one..

Reply:
since you are changing vlan of the interface, can it be checked via one of the below methods ?
1. command - ethtool
2. sysfs - /sys/class/net/interface/<vlan or something>

Reply:
@narasimhan-v here i will check for "Link detected" ..?

Reply:
Sure

Reply:
@narasimhan-v  but i checked with is_interface_link_up() from configure_network file,that is not working.that also checking for link up down through  /sys/class/net/interface.

Reply:
@smruti77 can you try to check /proc/net/vlan and see if that helps in removing this sleep. If even that does not work, let us go via sleep.

Reply:
@narasimhan-v i tried with ethtool also still its not working.

Reply:
Okay. Please mention all those in the comment.